test/Microsoft.AspNet.Mvc.Localization.Test/HtmlLocalizerTest.cs

```diff
@@ -150,9 +152,168 @@ public void HtmlLocalizer_HtmlWithInvalidResourcestring_ThrowsException(string f
             Assert.NotNull(exception);
             Assert.Equal("Input string was not in a correct format.", exception.Message);
         }
+
+        [Fact]
+        public void HtmlLocalizer_GetString_ReturnsLocalizedString()
+        {
+            // Arrange
+            var stringLocalizer = new TestStringLocalizer();
+
+            var htmlLocalizer = new HtmlLocalizer(stringLocalizer, new CommonTestEncoder());
+
+            // Act
+            var actualLocalizedString = htmlLocalizer.GetString("John");
+
+            // Assert
+            Assert.Equal("Hello John", actualLocalizedString.Value);
+        }
+
+        [Fact]
+        public void HtmlLocalizer_GetStringWithArguments_ReturnsLocalizedString()
+        {
+            // Arrange
+            var stringLocalizer = new TestStringLocalizer();
+
+            var htmlLocalizer = new HtmlLocalizer(stringLocalizer, new CommonTestEncoder());
+
+            // Act
+            var actualLocalizedString = htmlLocalizer.GetString("John", "Doe");
+
+            // Assert
+            Assert.Equal("Hello John Doe", actualLocalizedString.Value);
```

Comment:
You've added `Html()` tests but none with arguments. Think that should be covered with one test for `HtmlLocalizer` and one for `ViewLocalizer`.